src/editor/codemirror/language-server/documentation.ts

```diff
@@ -100,10 +101,24 @@ export const subsetMarkdown = (
   return sections.join("\n\n");
 };
 
+const createStyledAnchorElement = (): HTMLAnchorElement => {
+  const anchor = document.createElement("a");
+  anchor.href = "";
+  anchor.style.fontSize = "var(--chakra-fontSizes-sm)";
+  anchor.style.color = "var(--chakra-colors-brand-600)";
+  anchor.style.display = "block";
+  anchor.style.margin = "0";
+  anchor.style.marginRight = "-0.5rem";
```

Comment:
Please check this still makes sense.

Reply:
I can see the reasoning, but I think it's not necessary. Feels like the click target for the link is bigger without the negative margin as it's further in from the edge. [`5cb6a04`](https://github.com/microbit-foundation/python-editor-next/pull/806/commits/5cb6a040978f827858446176643dabd07a9897bf).